docs/DafnyRef/Introduction.md

```diff
@@ -26,12 +26,12 @@ checker—when the tool produces errors, the programmer responds by
 changing the program’s type declarations, specifications, and statements.
 
 (This document typically uses "Dafny" to refer to the programming language
-and "dafny" to refer to the software tool that verifies and compiles programs
+and `dafny` to refer to the software tool that verifies and compiles programs
 in the Dafny language.)
 
-The easiest way to try out Dafny is to [download](https://github.com/dafny-lang/dafny/releases) it
+The easiest way to try out `dafny` is to [download](https://github.com/dafny-lang/dafny/releases) it
```

Comment:
I understand this sentence as meaning that you try out the programming language, not the tool — hence the switch to `dafny` doesn't read too well

Reply:
I'll revise a bit in a later PR